dotcom-rendering/src/layouts/TagFrontLayout.tsx

```diff
@@ -213,6 +213,12 @@ export const TagFrontLayout = ({ tagFront, NAV }: Props) => {
 						groupedTrails.day !== undefined,
 					);
 
+					const imageLoading =
+						tagFront.config.abTests.lazyLoadImagesVariant ===
+							'variant' && index > 0
```

Comment:
Wondering what the benefit of eagerly loading the first container, does having the images set to Lazy cause issues?

Reply:
That’s what is recommended [in the Google article linked above](https://web.dev/browser-level-image-lazy-loading/).

<img width="754" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/76776/82b13b32-8eaa-44a6-b2d7-cb1c65206cee">

> You should avoid setting loading=lazy for any images that are in the first visible viewport. This is particularly relevant for LCP images. See the article [The performance effects of too much lazy-loading](https://web.dev/lcp-lazy-loading/) for more information.
> ([Source](https://web.dev/browser-level-image-lazy-loading/#avoid-lazy-loading-images-that-are-in-the-first-visible-viewport))